### PR TITLE
Abramtsova

### PR DESCRIPTION
**Pull request by totoshk**

### CODE REVIEW

File: src/components/Search/Search.js [hunk 1]
```diff
@@ -5,16 +5,25 @@ require('./search.scss');
...
+      </div>
+      <input
+        type="text"
+        name="search"
+        placeholder="Find a device"
+        className='searchBox__field'
+        onChange={e => {
+          props.handleSearch(e.target.value);
+        }}/>
+      <button className='btn btn--sucsess searchBox__btn'>Search</button>
```
Comment (SvitlanaAvry) [inline src/components/Search/Search.js]: The class name "btn-sucSess" is misspelled. This may cause errors.

**Review by SvitlanaAvry — APPROVED** [review]
+

**Review by SvitlanaAvry — APPROVED** [review]
+

**Review by SvitlanaAvry — APPROVED** [review]
+

**Review by SvitlanaAvry — APPROVED** [review]
+

File: src/components/Search/Search.js [hunk 2]
```diff
@@ -8,15 +8,17 @@ const Search = (props) => {
...
+      <div className='searchBox-wrap'>
+        <input
+          type="text"
+          name="search"
+          placeholder="Find a device"
+          className='searchBox__field'
+          onChange={e => {
+            props.handleSearch(e.target.value);
+          }}/>
+        <button className='btn btn--sucsess searchBox__btn'>Search</button>
```
Comment (SvitlanaAvry) [inline src/components/Search/Search.js]: The class name "btn--sucSess" is misspelled.

**Review by SvitlanaAvry — APPROVED** [review]
+

**Review by SvitlanaAvry — APPROVED** [review]
+

**Review by RostyslavSt — APPROVED** [review]
сool !

**Pull request merged**